src/Orc.NuGetExplorer.Xaml/Commands/PageBatchUpdatePackagesCommandContainer.cs

```diff
@@ -0,0 +1,130 @@
+﻿namespace Orc.NuGetExplorer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Catel;
+    using Catel.Logging;
+    using Catel.MVVM;
+    using Catel.Services;
+    using Orc.NuGetExplorer.Management;
+    using Orc.NuGetExplorer.Packaging;
+    using Orc.NuGetExplorer.Windows;
+
+    internal class PageBatchUpdatePackagesCommandContainer : CommandContainerBase<IManagerPage>
```

Comment:
Rename to ExtensionsBatchUpdatePackagesCommandContainer

Reply:
We have Orc.Extensibility for the Extensions things. NuGetExplorer is now using not only in the projects with Extensions.
My opinion is using "Extensions" word in Orc.NuGetExplorer is not a good idea